media-libs/panda3d/panda3d-1.9.2.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 2008-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python{2_7,3_4,3_5} )
+
+inherit eutils multiprocessing python-single-r1
+
+if [[ ${PV} != 9999 ]]; then
+	MY_PV=v${PV}
+	KEYWORDS="~amd64 ~x86"
+	SRC_URI="http://www.panda3d.org/download/${P}/${P}.tar.gz"
+	S="${WORKDIR}"/${P}
+else
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/panda3d/panda3d.git"
+	EGIT_BRANCH="master"
+fi
+
+DESCRIPTION="A 3D game engine and framework for Python and C++."
+HOMEPAGE="http://www.panda3d.org"
+
+LICENSE="BSD"
+SLOT="0"
+
+IUSE="bullet +eigen ffmpeg fftw fmod jpeg opencv png +python +openal +ssl tiff truetype zlib"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="
+	bullet? ( sci-physics/bullet )
+	eigen? ( dev-cpp/eigen:3 )
+	ffmpeg? ( media-video/ffmpeg )
+	fmod? ( media-libs/fmod:= )
```

Comment:
changed to slot 1 as the only in-tree version of fmod with slot 0 is x86-only and according to panda commit 8403e175 from 2009 they supported 4.22.something then (4.x is slot 1)